tests/integration/test_storage_iceberg/test.py

```diff
@@ -423,6 +425,15 @@ def test_multiple_iceberg_files(started_cluster, format_version, storage_type):
         "SELECT number, toString(number + 1) FROM numbers(200)"
     )
 
+    assert (
```

Comment:
Firstly, please, create a new test (you can do it directly in this file). Secondly, we need better testing here. You can insert several blocks with some lag and verify the table state between these insertions